vara/conn.go

```diff
@@ -95,7 +96,7 @@ func (v *conn) Close() error {
 			debugPrint("close: discarded %d bytes of remaining data", n)
 		}()
 		v.closing = true
-		connectChange, cancel := v.cmds.Subscribe(disconnected)
+		connectChange, cancel := v.cmds.Subscribe("DISCONNECTED")
 		defer cancel()
 		if v.connectedState == disconnected {
```

Comment:
Should this also be changed to a literal string?

Reply:
Same, PR description makes it clear this is supposed to be the internal representation.